src/async_jobs.ts

```diff
@@ -243,6 +259,52 @@ async function slackInteractivityHandler(
         return;
       }
 
+      case SlackCallbackId.ROUTE_TO_JOURNEY: {
+        const { viewId } = interactivityMetadata;
+        if (!viewId) {
+          throw new Error(
+            'slackInteractivityHandler called for message_action without viewId'
+          );
+        }
+
+        const MD5 = new Hashes.MD5();
+
+        // Ignore Prettier formatting because this object needs to adhere to JSON strigify requirements.
```

Comment:
I'm not sure what this means -- Prettier doesn't change the semantics of your code; there shouldn't be any runtime difference between prettier code and non-prettier code.

Reply:
This is copied from RESET_DEMO and it's been a while but IIRC the double quotes are necessary for the keys (prettier removes them) in order for JSON.stringify to work, for some reason. I can look up the documentation I found previously again if necessary.

Reply:
I would give this another try- the quotes around the key names are just used during parsing when the code is loaded; they're not going to affect how the code works.